config/passport.js

```diff
@@ -0,0 +1,36 @@
+const passport = require('passport');
+const JwtStrategy = require('passport-jwt').Strategy;
+const { PRIVATE_KEY } = require('../config/apiCredentials');
+
+const cookieExtractor = (request) => {
+	var token = null;
+
+	if (request && request.headers.cookie) {
+		token = request.headers.cookie.split('=')[1];
```

Comment:
What would happen here in the future if we use cookies for other reasons? Would this always work?

(And incidentally, there's an instance of a magic number here, but fixing my main question will hopefully end up removing the magic number anyway)

Reply:
(I believe there's a library mentioned on the passport docs that's quite useful, to do with cookies)

Reply:
Added cookie-parser and updated code to use this.